framework/architecture/s3/s3_properties.rst

```diff
@@ -0,0 +1,46 @@
+.. _s3_architecture:
+
+Architecture
+============
+
+
+S3 Integration Diagram only for Developer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An overlay bucket is provided for the developers.
+This prevents developers from writing data to the productive bucket during testing.
+
+
+The S3 datastorage is created for Tocco with the following buckets.
+
+1. A bucket for the client
+2. A bucket for the developers, which is overwritten due to configuration.
+
+
+
+.. image:: ./resources/s3_architecture.jpg
+    :width: 600pt
+
+
+S3 Integration details
+^^^^^^^^^^^^^^^^^^^^^^
+
+In the Tocco framework, the method ``createClient()`` is called from the file
```

Comment:
In the Tocco framework, ~~the method~~ ``createClient()`` is called from ~~the file~~

I like conciseness.